.gitignore

```diff
@@ -3,4 +3,3 @@ output/
 assets/docs/latest/en/
 stuff/
 .DS_Store
-output
```

Comment:
think this should stay, because those files should not be committed, no?

Reply:
It's a duplicate - there's another line further up the file already.

Reply:
😮